content_blocker_update.sh

```diff
@@ -4,15 +4,26 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+# Current shavar version
+CURRENT_SHAVAR_LIST_VERSION="107"
```

Comment:
We can start by just adding the changes in this script as a first step. We will see how it works while we wait for the shavar questions to be answered to move forward.